HISTORY.md

```diff
@@ -2,6 +2,7 @@
 ## Unreleased
 ### Bug Fixes
 * Fix a bug in io_uring_prep_cancel in AbortIO API for posix which expects sqe->addr to match with read request submitted and wrong paramter was being passed.
+* Fixed a regression in iterator performance when the entire DB is a single memtable.
```

Comment:
It would be good to mention both the PR introducing this and the PR fixing this regression.